source-code/ide-extension/src/diagnostics/linterDiagnostics.ts

```diff
@@ -0,0 +1,99 @@
+import { getLintReports, lint, LintReport } from "@inlang/core/lint"
+import * as vscode from "vscode"
+import { state } from "../state.js"
+
+export async function linterDiagnostics(args: { context: vscode.ExtensionContext }) {
+	const linterDiagnosticCollection = vscode.languages.createDiagnosticCollection("inlang-lint")
+
+	async function updateLintDiagnostics() {
+		const activeTextEditor = vscode.window.activeTextEditor
+		if (!activeTextEditor) {
+			return
+		}
+
+		const resources = state().resources
+
+		const [resourcesWithLints, errors] = await lint({ resources, config: state().config })
+
+		if (errors) {
+			console.log(errors)
```

Comment:
Do this errors include errors which are of interest for the ide-extension user?

Reply:
Let me check 👍

Reply:
The errors should not be user-facing for now.